### DISCUSSION TITLE
constant issue while installing foundry on windows

### DISCUSSION THREAD

**Discussion by kwadboom** [Cyfrin/foundry-full-course-cu#2200] [Q&A]
![foundry ](https://github.com/user-attachments/assets/6d0252d9-6438-43a5-a759-4dc6c19caae4)
curl -L https://foundry.paradigm.xyz | bash
% total % received % xferd average speed  time  time  time  current 
                           dload   upload total spent left  speed
100 167 100 167    0 0    268      0     --:-- --:-- --:--  268
0    0   0   0     0 0     0       0     --:-- --:-- --:--   0
curl: (28) SSL connection timeout

**Comment by EngrPips** [comment]
Hello @kwadboom, Why are you opening multiple discussions for a single issue?
